source/customization.rst

```diff
@@ -252,9 +252,16 @@ We recommend setting this environment variable in ``/etc/ood/config/nginx_stage.
 Set Default SSH Host
 --------------------
 
-In ``/etc/ood/config/apps/shell/env`` set the env var ``DEFAULT_SSHHOST`` to change the default ssh host. Otherwise it will default to "localhost" i.e. add the line ``DEFAULT_SSHHOST="localhost"``.
+In ``/etc/ood/config/apps/shell/env`` set the env var ``OOD_DEFAULT_SSHHOST`` to change the default ssh host. Otherwise it will default to the first cluster yaml file with ``default: true`` under login. Else it will default to the first cluster yaml file that does not have ``hidden: true``
```

Comment:
I'll have to dig into this more, but I think that there is no base out of the box default anymore.

If so, we should add that it no longer will default to `localhost`. You have to set it somewhere through the environment variables or through cluster yamls.

Reply:
Yeah, call out that in OnDemand 1.8 and later this will no longer default to localhost.

Reply:
Also the sentences don't read well. Maybe:

>In ``/etc/ood/config/apps/shell/env`` set the env var ``OOD_DEFAULT_SSHHOST`` to specify the default ssh host. If not set, the default login host will be the that of the first cluster config file that has ``default: true`` in the login section. If no cluster config specifies ``default: true``, the first cluster config that does not have ``hidden: true`` in the meta section be used.